gnocchi/carbonara.py

```diff
@@ -564,9 +564,12 @@ def split(self):
     @classmethod
     def from_timeseries(cls, timeseries, sampling, aggregation_method,
                         max_size=None):
```

Comment:
Perhaps we need to add a comment to say timeseries parameter don't allow overlap anymore.